fix: Update last criterion on MFA tools page (#2618)
privacyguides/privacyguides.org: docs/multi-factor-authentication.md

- Source code must be publicly available.
- Must not require internet connectivity.

> - Must not sync to a third-party cloud sync/backup service by default.
>     - **Optional** E2EE sync support is acceptable.

Reviewer: ```suggestion
- Cloud syncing must be optional, and (if available) sync functionality must be E2EE.
```
